packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -550,16 +556,31 @@ export const ChatMessage: ComponentWithAs<'div', ChatMessageProps> &
       }),
     });
 
+    const bodyElement = Box.create(comfyBody || {}, {
+      defaultProps: () =>
+        getA11Props('comfyBody', {
+          className: chatMessageSlotClassNames.comfyBody,
+          styles: resolvedStyles.comfyBody,
+        }),
+      overrideProps: () => ({
+        content: (
+          <>
+            {actionMenuElement}
+            <div className={chatMessageSlotClassNames.bar} />
```

Comment:
it might be necessary to create a slot for this instead of div so that we can target it with potential props, not just styles

Reply:
oh I can see that this is just moved code, in that case let's just leave it as is